src/helpers.ts

```diff
@@ -1,8 +1,50 @@
+const fontWeightKeywords = require('css-font-weight-keywords');
 const cssFontSizeKeywords = require('css-font-size-keywords');
+const fontStretchKeywords = require('css-font-stretch-keywords');
+
+const sizePattern = /^\+?[\d\.]/;
+const numberPrefixPattern = /^(\+|-)?(\.)?\d/;
+const percentPattern = /%$/;
 
 export function isSize(value: string) {
-	return /^[\d\.]/.test(value)
+	return sizePattern.test(value)
 		|| value.indexOf('/') !== -1
 		|| cssFontSizeKeywords.indexOf(value) !== -1
 	;
 }
+
+export function isWeight(value: string) {
+	return fontWeightKeywords.indexOf(value) !== -1
+		|| isOnlyNumber(value);
+}
+
+export function isStretch(value: string) {
+	return fontStretchKeywords.indexOf(value) !== -1
+		|| isOnlyPercentNumber(value);
+}
+
+function isOnlyPercentNumber(value: string): boolean {
+	if (percentPattern.test(value)) {
+		return isOnlyNumber(value.substring(0, value.length - 1));
+	}
+	return false;
+}
+
+function isOnlyNumber(value: string): boolean {
```

Comment:
[`parse-css-dimension`](https://github.com/jedmao/parse-css-dimension) might also be useful here.

Reply:
I am not sure:
the "allow number only" parsing is not really provided by `parse-css-dimension`, or more precisely, it does not make sure that the input-string does not contain more than the number and/or the the unit

E.g. for `"43asdfa7rem"` it will return a valid result with number `43` and unit `"rem"`

Reply:
Aha. Sounds like it needs a strict mode then. That can be arranged.

Reply:
yes, then most of this PR should be removed (or moved to `parse-css-dimension` where appropriate) & replaced by using `parse-css-dimension`

Would you start on `parse-css-dimension`, e.g. how it should be invoked for "strict mode"?
e.g. maybe something like `parseCssDimension('42dpi', 'strict')` or maybe `parseCssDimension('42dpi', {strict: true})`, or ...?

I would then transfer/add parts from this PR for the strict-number testing as much as possible

Reply:
I think we should go with an options hash of `{ strict: true }`.

Reply:
@russaa see https://github.com/jedmao/parse-css-dimension/issues/4. Also, [I'm updating that project to TypeScript/AVA](https://github.com/jedmao/parse-css-dimension/pull/3) like this one. Feel free to submit a PR for https://github.com/jedmao/parse-css-dimension/issues/4.